test/adapter/json_api/parse_test.rb

```diff
@@ -0,0 +1,67 @@
+require 'test_helper'
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        module Deserialization
+          class ParseTest < Minitest::Test
+            def setup
```

Comment:
Needs a 'reversible' test.

1. Given a resource
2. Serialize resource to hash
3. deserialize to resource (attributes?)